src/reformat.c

```diff
@@ -1241,8 +1241,11 @@ static avifResult avifRGBImageToF16(avifRGBImage * rgb)
 static avifResult avifImageYUVToRGBImpl(const avifImage * image, avifRGBImage * rgb, avifReformatState * state, avifAlphaMultiplyMode alphaMultiplyMode)
 {
     avifBool convertedWithLibYUV = AVIF_FALSE;
+    avifBool reformatAlpha = avifRGBFormatHasAlpha(rgb->format) &&
+                             (!rgb->ignoreAlpha || (alphaMultiplyMode != AVIF_ALPHA_MULTIPLY_MODE_NO_OP));
+    avifBool alphaReformattedWithLibYUV = AVIF_FALSE;
```

Comment:
Optional: Document `alphaReformattedWithLibYUV` is only checked when `reformatAlpha` is true? Okay if this is obvious.